activation/post_supervisor_win.go

```diff
@@ -0,0 +1,29 @@
+//go:build windows
+
+package activation
+
+import (
+	"os/exec"
+	"path/filepath"
+)
+
+// DefaultPostServiceConfig returns the default config for post service.
+func DefaultPostServiceConfig() PostSupervisorConfig {
+	return PostSupervisorConfig{
+		PostServiceCmd: "C:\\service.exe",
```

Comment:
Why this location? Perhaps the default should be "not set"?

Reply:
I updated the default configs to look for `service` and `service.exe` in the same folder as the node. This also reduced the build flags for windows and *nix to just a constant that is different for the two systems.